src/contracts/scripts/Task0004.s.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {ArbScript} from "scripts/utils/ArbScript.s.sol";
+import {Help, Log} from "kresko-lib/utils/Libs.s.sol";
+import {SwapRouteSetter} from "scdp/STypes.sol";
+import {scdp} from "scdp/SState.sol";
+import "scripts/deploy/JSON.s.sol" as JSON;
+import {ArbDeployAddr} from "kresko-lib/info/ArbDeployAddr.sol";
+import {deployPayload} from "scripts/payloads/Payloads.sol";
+import {LibDeploy} from "scripts/deploy/libs/LibDeploy.s.sol";
+import {LibJSON} from "scripts/deploy/libs/LibJSON.s.sol";
+import {IExtendedDiamondCutFacet} from "diamond/interfaces/IDiamondCutFacet.sol";
+import {Payload0004} from "scripts/payloads/Payload0004.sol";
+import {IVault} from "vault/interfaces/IVault.sol";
+// solhint-disable no-empty-blocks, reason-string, state-visibility
+
+contract Task0004 is ArbScript {
+    using Log for *;
+    using Help for *;
+    using LibDeploy for JSON.Config;
+    using LibDeploy for bytes;
+    using LibJSON for JSON.Config;
+    using LibJSON for JSON.AssetJSON;
+    uint256 currentForkId;
+
+    function payload0004() public {
+        if (currentForkId == 0) {
```

Comment:
```suggestion
        ArbScript.initialize();
        if (currentForkId == 0) {
```

This is a bit annoying but we have to use this initialize to apply few workarounds, mainly here is the setting of DeploymentFactory address

Reply:
Fixed in e174a2f